files/simple-statistics/update.json

```diff
@@ -0,0 +1,9 @@
+{
+  "packageManager": "github",
+  "name": "simple-statistics",
+  "repo": "simple-statistics/simple-statistics",
+  "files": {
+    "include": ["simple-statistics.js", "simple-statistics.min.js"],
```

Comment:
The file names are not correct. The repo has them as `simple_statistics.min.js`
Also feel free to remove the unminified file, its not needed in production.